_scss/tutorial/highlight_block.scss

```diff
@@ -22,20 +21,17 @@
 }
 
 @media (min-width: 480px) {
-.steps {
-    .highlight {
-       width: 50%;
-       float: left;
-       max-width: 600px;
-       margin-left: auto;
-       margin-right: auto;
-       margin-bottom: -500px;
-       padding-bottom: 500px;
-    }
-  }
 }
 
 @media (min-width: 768px) {
+  .steps {
+      .highlight {
+         width: 50%;
+         float: left;
+         margin-bottom: -500px;
+         padding-bottom: 500px;
+      }
+    }
 }
```

Comment:
This looks great and the highlight div is 100% now - except that for some reason the background color of the `.list` class is overwriting the black of the `.highlight` class. It's super weird since highlight blocks aren't nested inside list .. :-/ 

<img width="200" alt="screen shot 2017-01-06 at 10 24 28 am" src="https://cloud.githubusercontent.com/assets/6585221/21722497/e953b9c0-d3fa-11e6-9adc-8b478d757785.png">